lib/membrane_h264_ffmpeg/encoder.ex

```diff
@@ -72,6 +82,23 @@ defmodule Membrane.H264.FFmpeg.Encoder do
                 spec: H264.profile_t() | nil,
                 default: nil
               ],
+              tune: [
+                description: """
+                To change settings based upon the specifics of input. For example, if your input is animation then use the animation tuning
+                or if you want to preserve grain in a film then use the grain tuning. If you are unsure of what to use or your input does not match any of tunings
+                then omit the tune option. Available options are:
+                  :film - use for high quality movie content; lowers deblocking
+                  :animation - good for cartoons; uses higher deblocking and more reference frames
+                  :grain - preserves the grain structure in old, grainy film material
+                  :stillimage - good for slideshow-like content
+                  :fastdecode - allows faster decoding by disabling certain filters
+                  :zerolatency - good for fast encoding and low-latency streaming
+                  :psnr - ignore this as it is only used for codec development
+                  :ssim - ignore this as it is only used for codec development
```

Comment:
Format this properly, i.e.
1. Make it a markdown list
2. Wrap atoms with backticks